lib/board.rb

```diff
@@ -0,0 +1,50 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" "," "," "," "," "," "," "," "," ",]
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} \n"
+    puts "-----------\n"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} \n"
+    puts "-----------\n"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def reset!
+    self.cells = [" "," "," "," "," "," "," "," "," ",]
+  end
+
+  def position(input)
+    index = (input.to_i-1)
+    self.cells[index]
+  end
+
+  def full?
+    !self.cells.include?(" ") || false
```

Comment:
Use the most expressive and efficient iterator for a problem. `Board#full?` is asking if all elements are not empty strings. You can use logic that is symmetrical to those semantics with the [`#all?`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-all-3F) iterator.

`cells.all? {|cell| cell != " "}`

This is also more efficient in that the loop will stop after the iteration for which a cell is not empty.